app/shared_features/edit_set/EditTextModal.js

```diff
@@ -172,7 +172,7 @@ class EditTextModal extends Component {
                 { statusBar }
 
                 <View style={{position: 'absolute', left: 0, top: 0}}>
-                    <TouchableOpacity onPress={() => this.props.closeModal()}>
+                    <TouchableOpacity onPress={() => this.props.cancelModal(this.state.setID)}>
```

Comment:
I don't mind this, but in that case the SAVE actions should also cause the closeModal rather than having a closeModal as a separate action

Reply:
Backlog